code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -174,6 +174,42 @@
 	body_parts_covered = CHEST|ARMS|GROIN|LEGS|FEET
 	flags_inv = HIDESHOES|HIDEJUMPSUIT
 
+//I will ensure that wearing this will warent a sufficient beating to take it off again
+/obj/item/clothing/head/chungus
+	name = "chungus hat"
+	desc = "BIG CHUNGUS HUER DU UUUUUUUUUUUUUUUU CHUNGUS FUNNY REDDIT MOMENT BIG GROBE CHUNGUS KIONO REEVES WHOLE SOME 100 ELON MUSK."
+	icon_state = "bigchungushead"
+	armor = list("melee" = -50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 100, "rad" = 0, "fire" = 0, "acid" = 100 ,"stamina" = 100)
+	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDEFACIALHAIR
+	clothing_flags = SNUG_FIT
+
+/obj/item/clothing/suit/chungus
+	name = "chungus suit"
+	desc = "BIG CHUNGUS HUER DU UUUUUUUUUUUUUUUU CHUNGUS FUNNY REDDIT MOMENT BIG GROBE CHUNGUS KIONO REEVES WHOLE SOME 100 ELON MUSK."
+	icon_state = "chungus"
+	armor = list("melee" = -50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 100, "rad" = 0, "fire" = 0, "acid" = 100 ,"stamina" = 100)
+	body_parts_covered = CHEST|ARMS|GROIN|LEGS|FEET
+	flags_inv = HIDESHOES|HIDEJUMPSUIT
+
+/obj/item/clothing/suit/chungus/equipped(mob/user, slot)
+	. = ..()
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		H.gain_trauma(/datum/brain_trauma/severe/discoordination,TRAUMA_RESILIENCE_ABSOLUTE)
+		H.gain_trauma(/datum/brain_trauma/mild/stuttering,TRAUMA_RESILIENCE_ABSOLUTE)
+		H.gain_trauma(/datum/brain_trauma/mild/dumbness,TRAUMA_RESILIENCE_ABSOLUTE)
+		H.gain_trauma(/datum/brain_trauma/mild/speech_impediment,TRAUMA_RESILIENCE_ABSOLUTE)
+		H.gain_trauma(/datum/brain_trauma/mild/expressive_aphasia,TRAUMA_RESILIENCE_ABSOLUTE)
+
+/obj/item/clothing/suit/chungus/dropped(mob/user)
+	. = ..()
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		H.cure_trauma_type(/datum/brain_trauma/severe/discoordination,TRAUMA_RESILIENCE_ABSOLUTE)
+		H.cure_trauma_type(/datum/brain_trauma/mild/stuttering,TRAUMA_RESILIENCE_ABSOLUTE)
+		H.cure_trauma_type(/datum/brain_trauma/mild/dumbness,TRAUMA_RESILIENCE_ABSOLUTE)
+		H.cure_trauma_type(/datum/brain_trauma/mild/speech_impediment,TRAUMA_RESILIENCE_ABSOLUTE)
+		H.cure_trauma_type(/datum/brain_trauma/mild/expressive_aphasia,TRAUMA_RESILIENCE_ABSOLUTE)
```

Comment:
You should not use cure_trauma_type for curing a trauma related to a specific atom, especially absolute traumas. If they had the trauma already, then picking up and dropping the item would heal the trauma. See /datum/reagent/drug/bath_salts for a better way to do this.